.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,21 +1,17 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: bug
+about: Create a bug report to help us improve
+title: "[Bug]: "
```

Comment:
Why do we need to prefix the title if it already has a label set?

Reply:
Good point, it was common on many examples I looked at but it seems to just be some flavor.